aiogram/fsm/storage/base.py

```diff
@@ -19,6 +19,68 @@ class StorageKey:
     destiny: str = DEFAULT_DESTINY
 
 
+class KeyBuilder(ABC):
+    """Base class for key builder."""
+
+    @abstractmethod
+    def build(self, key: StorageKey, part: Literal["data", "state", "lock"]) -> str:
+        """
+        This method should be implemented in subclasses
```

Comment:
Redundant comment, `@abstractmethod` already meant this.
Use imperative method description only.